<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QJournalctl</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/ui/icon.xpm</normaloff>:/ui/icon.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>QJournalctl</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="showCurrentBootLogButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Show the current boot log</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="showBootLogButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Show boot log</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="3">
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>2</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="listBootsButton">
      <property name="text">
       <string>List all available boots</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="reverseCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Reverse output (latest message on top)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="realtimeCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Follow the journal in realtime</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJournalctl_Analyzer_Demo">
    <property name="title">
     <string>&amp;journalctl</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionShowCurrentBootLog"/>
    <addaction name="actionLoadBoots"/>
    <addaction name="actionShowCompleteJournal"/>
    <addaction name="separator"/>
    <addaction name="actionSizeOfTheJournalOnTheDisk"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOK">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuRemote">
    <property name="title">
     <string>Remote</string>
    </property>
    <addaction name="actionOpen_a_new_SSH_connection"/>
    <addaction name="separator"/>
    <addaction name="actionLast_Connection_1"/>
    <addaction name="actionLast_Connection_2"/>
    <addaction name="actionLast_Connection_3"/>
    <addaction name="actionLast_Connection_4"/>
   </widget>
   <addaction name="menuJournalctl_Analyzer_Demo"/>
   <addaction name="menuRemote"/>
   <addaction name="menuOK"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadBoots">
   <property name="text">
    <string>&amp;List all available boots</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionShowCompleteJournal">
   <property name="text">
    <string>&amp;Show the complete journal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSizeOfTheJournalOnTheDisk">
   <property name="text">
    <string>Size &amp;of the journal on the disk</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionShowCurrentBootLog">
   <property name="text">
    <string>Show the &amp;current boot log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionOpen_a_new_SSH_connection">
   <property name="text">
    <string>Open a new SSH connection</string>
   </property>
  </action>
  <action name="actionLast_Connection_1">
   <property name="text">
    <string>Last Connection #1</string>
   </property>
  </action>
  <action name="actionLast_Connection_2">
   <property name="text">
    <string>Last Connection #2</string>
   </property>
  </action>
  <action name="actionLast_Connection_3">
   <property name="text">
    <string>Last Connection #3</string>
   </property>
  </action>
  <action name="actionLast_Connection_4">
   <property name="text">
    <string>Last Connection #4</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>ListBoots()</slot>
 </slots>
</ui>
